tests/test_embeds.py

```diff
@@ -479,4 +479,4 @@ def test_emptyembed() -> None:
 
     # make sure unknown module attrs continue to raise
     with pytest.raises(AttributeError):
-        embeds.this_does_not_exist  # type: ignore
+        embeds.this_does_not_exist  # type: ignore  # noqa: B018
```

Comment:
Instead of adding `noqa`s, `_ = embeds.this_does_not_exist` would make this explicit

Reply:
[`a071fd6` (#999)](https://github.com/DisnakeDev/disnake/pull/999/commits/a071fd6f15a8b5b540a7e605b46f99cd23ac86c6)